src/MLJMultivariateStatsInterface.jl

```diff
@@ -150,4 +150,1078 @@ metadata_pkg.(
+
+# Fitted parameters
+
+The fields of `fitted_params(mach)` are:
+
+- `coefficients`: The linear coefficients determined by the model.
+- `intercept`: The intercept determined by the model.
+
+# Examples
+
+```
+using MLJ
+
+LinearRegressor = @load LinearRegressor pkg=MultivariateStats
+linear_regressor = LinearRegressor()
+
+X, y = make_regression(100, 2) # synthetic data
+mach = machine(linear_regressor, X, y) |> fit!
+
+Xnew, _ = make_regression(3, 2)
+yhat = predict(mach, Xnew) # new predictions
+```
+
+See also
+[`MultitargetLinearRegressor`](@ref), [`RidgeRegressor`](@ref), [`MultitargetRidgeRegressor`](@ref)
+"""
+LinearRegressor
+
+"""
+$(MMI.doc_header(MultitargetLinearRegressor))
+
+`MultitargetLinearRegressor` assumes the target variable is vector-valued with
+continuous components.  It trains a linear prediction function using the
+least squares algorithm. Options exist to specify a bias term.
+
+# Training data
+
+In MLJ or MLJBase, bind an instance `model` to data with
+
+    mach = machine(model, X, y)
+
+Where
+
+- `X`: is any table of input features (eg, a `DataFrame`) whose columns
+  are of scitype `Continuous`; check the scitypes with `schema(X)`
+
+- `y`: is the target, which can be any table of responses whose element
+  scitype is `Continuous`; check the scitype with `scitype(y)`
+
+Train the machine using `fit!(mach, rows=...)`.
+
+# Hyper-parameters
+
+- `bias=true`: Include the bias term if true, otherwise fit without bias term.
+
+# Operations
+
+- `predict(mach, Xnew)`: Return predictions of the target given new
+  features `Xnew` having the same scitype as `X` above.
+
+# Fitted parameters
+
+The fields of `fitted_params(mach)` are:
+
+- `coefficients`: The linear coefficients determined by the model.
+- `intercept`: The intercept determined by the model.
+
+# Examples
+
+```
+using MLJ
+using MLJBase: augment_X
+using DataFrames
+
+LinearRegressor = @load MultitargetLinearRegressor pkg=MultivariateStats
+linear_regressor = LinearRegressor()
+
+X = augment_X(randn(100, 8), true)
+θ = randn((9,2))
+y = X * θ
+X, y = map(x -> DataFrame(x, :auto), (X, y))
+
+mach = machine(linear_regressor, X, y) |> fit!
+
+Xnew, _ = make_regression(3, 9)
+yhat = predict(mach, Xnew) # new predictions
+```
+
+See also
+[`LinearRegressor`](@ref), [`RidgeRegressor`](@ref), [`MultitargetRidgeRegressor`](@ref)
+"""
+MultitargetLinearRegressor
+
+"""
+$(MMI.doc_header(RidgeRegressor))
+
+`RidgeRegressor` adds a quadratic penalty term to least squares regression,
+for regularization. Ridge regression is particularly useful in the case of
+multicollinearity.
+Options exist to specify a bias term, and to adjust the strength of the penalty term.
+
+# Training data
+
+In MLJ or MLJBase, bind an instance `model` to data with
+
+    mach = machine(model, X, y)
+
+Where
+
+- `X`: is any table of input features (eg, a `DataFrame`) whose columns
+  are of scitype `Continuous`; check the scitypes with `schema(X)`
```

Comment:
Sorry I missed this earlier. Please change everywhere - I'm not going to mark each one.

```suggestion
  are of scitype `Continuous`; check column scitypes with `schema(X)`
```